.mocharc.json

```diff
@@ -1,8 +1,8 @@
 {
-    "require": "ts-node/register,source-map-support/register",
-    "watch-extensions": "ts",
-    "recursive": true,
-    "reporter": "spec",
-    "timeout": 5000
-  }
-  
\ No newline at end of file
+  "require": "ts-node/register,source-map-support/register",
```

Comment:
was this updated by `dev-scripts`?

Reply:
no, that's me.  I like to run `yarn mocha foo --watch`

If all our stuff was more standardized we probably could fix the `watch` property, but there's enough variations in test-naming (ex: sdr) and paths (toolbelt) etc.